prep-javascript-loops/main.js

```diff
@@ -0,0 +1,66 @@
+function whileLoop1() {
+  const array = [];
+  let i = 0;
+  while (i < 10) {
+    array.push(i);
+    i++;
+  }
+  return array;
+}
+console.log('whileLoop1 output', whileLoop1());
+
+function whileLoop2() {
+  const array = [];
+  let i = 0;
+  while (i < 19) {
+    array.push(i);
+    i += 2;
+  }
+  return array;
+}
+console.log('whileLoop2 output', whileLoop2());
+
+function forLoop1() {
+  const array = [];
+  for (let i = 0; i < 10; i++) {
+    array.push(i);
+  }
+  return array;
+}
+console.log('forLoop1 output', forLoop1());
+
+function forLoop2() {
```

Comment:
for this function, you don't want to return an array, although you did that well! You just want to console log the string each time the loop decrements with the new index value.